scripts/update-api-tokens.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
Could we add an overview at the top of the file to explain what this does please without someone needing to actually scroll through the code

Reply:
Could also be good to rename this script something more explicit about how this creates prs rather than actually rolling credentials.

Reply:
👍